ament_package/__init__.py

```diff
@@ -123,7 +123,7 @@ def parse_package_string(data, *, filename=None):
         "Unable to handle '%s' format version '%d', please update the " \
         'manifest file to at least format version 2' % \
         (filename, pkg.package_format)
-    assert pkg.package_format in [2], \
+    assert pkg.package_format in [2, 3], \
```

Comment:
It would be nice to have some kind of test for this function (`parse_package_string()`), and have it updated for the new features in `package.xml` version 3. I guess those didn't make it over from `catkin_pkg`.